packages/apputils-extension/src/index.ts

```diff
@@ -422,7 +422,10 @@ const state: JupyterFrontEndPlugin<IStateDB> = {
         delete query['reset'];
 
         const url = path + URLExt.objectToQueryString(query) + hash;
-        const cleared = db.clear().then(() => router.stop);
+        const cleared = db
+          .clear()
+          .then(() => save.invoke())
+          .then(() => router.stop);
```

Comment:
I'm guessing this save turned out to be redundant, if you wanted to clean that up :)